Runtime/Scripts/RGBotServerListener.cs

```diff
@@ -827,11 +835,18 @@ private void HandleClientRequest(TcpClient client, uint clientId, string token,
         // call me on the main thread only
         private void HandleAction(uint clientId, RGActionRequest actionRequest)
         {
-            var agent = agentMap[clientId];
-            RGAction actionHandler = agent.GetActionHandler(actionRequest.action);
-            if (actionHandler != null)
+            var agents = agentMap[clientId];
+            /* TODO: Right now this broadcasts the action to all agents.  We may
+             * want to assign actions to specific agents based on their entity Id
+             * if we ever support more than 1 entity + button clicks
+             */
+            foreach (var agent in agents)
             {
-                actionHandler.StartAction(actionRequest.input);
+                RGAction actionHandler = agent.GetActionHandler(actionRequest.action);
+                if (actionHandler != null)
+                {
+                    actionHandler.StartAction(actionRequest.input);
```

Comment:
If multiple bots can receive the action given, do all take that action? I feel like that's fine, but just wanted to make sure I understand.

Also one more question - on the unity.js, there is still only "1 bot", correct? It may just happen to own multiple agents?